### PR TITLE
Fix/improve grammar in automatic.rst

### PR DESCRIPTION
**Pull request by kiates**
Mostly small grammar changes, but a few other adjustments related to flow.

* I've followed [this guide](https://docs.hunter.sh/en/latest/creating-new/create/cmake.html)
  step by step carefully. **[Yes|No]**

* I've tested this package remotely and have excluded all broken builds.
  Here is the links to the Travis/AppVeyor with status "All passed":

  * https://ci.appveyor.com/project/<username>/hunter/build/<build-number>
  * https://travis-ci.org/<username>/hunter/builds/<build-number>

* I have submitted CI configs to https://github.com/ingenue/hunter targeting `pkg.template` branch,
  see this merged pull request https://github.com/ingenue/hunter/pull/<number>

* My change will resolve this "New package" request: https://github.com/ruslo/hunter/issues/<number>

---

* I've followed [this guide](https://docs.hunter.sh/en/latest/creating-new/update.html)
  step by step carefully. **[Yes|No]**

* I've tested this package remotely and have excluded all broken builds.
  Here is the links to the Travis/AppVeyor with status "All passed":

  * https://ci.appveyor.com/project/<username>/hunter/build/<build-number>
  * https://travis-ci.org/<username>/hunter/builds/<build-number>

* This update will break few old toolchains.
  They are excluded in this pull request: https://github.com/ingenue/hunter/pull/<number>

---

* I've checked this [Git style guide](https://0.readthedocs.io/en/latest/git.html). **[Yes|No]**
* I've checked this [CMake style guide](https://0.readthedocs.io/en/latest/cmake.html). **[Yes|No]**
* My change will work with CMake 3.2 (minimum requirement for Hunter). **[Yes|No]**
* I will try to keep this pull request as small as possible and will try not to mix unrelated features. **[Yes|No]**

---

### CODE REVIEW

File: docs/overview/automatic.rst [hunk 1]
```diff
@@ -70,17 +64,23 @@ instead of:
...
 
-Vs. 'requirements.txt' style
+It's simply:
+
+::
+
+  > cmake -H. -B_builds -DBUILD_WITH_BAR=YES # + install bar
+
+Compared to a 'requirements.txt' style approach
 ============================
```
Comment (ruslo) [inline docs/overview/automatic.rst]: Need to add few `=` here.

File: docs/overview/automatic.rst [hunk 2]
```diff
@@ -91,7 +91,7 @@ file and there is user's option:
     command_to_install(Bar)
   endif()
 
-In case it's not CMake code this will look even fancy, say ``requirements.json``:
+Or, in the case that it isn't CMake code, this might by something fancy like ``requirements.json``:
```
Comment (ruslo) [inline docs/overview/automatic.rst]: Spell check doesn't like `isn't`. This is kind of a spell check limitation, you can add `isn` to `spelling.txt` or use `is not` here.

File: docs/overview/automatic.rst [hunk 3]
```diff
@@ -125,8 +125,8 @@ You have to repeat same condition in ``CMakeLists.txt`` file:
     target_compile_definitions(... PUBLIC "WITH_BAR")
   endif()
 
-In case when this part will change you must not to forget to modify ``requirements``
-accordingly too. And real world libraries can have nontrivial chain of conditions, e.g.
+Later, when you need to change this dependency in 'CMakeLists.txt', you'd better not forget to also modify ``requirements.cmake``
```
Comment (ruslo) [inline docs/overview/automatic.rst]: Use
```
``CMakeLists.txt``
```

File: docs/overview/automatic.rst [hunk 4]
```diff
@@ -70,17 +64,23 @@ instead of:
...
+
+::
+
+  > cmake -H. -B_builds -DBUILD_WITH_BAR=YES # + install bar
+
+Compared to a 'requirements.txt' style approach
+===============================================
 
-Note that such approach differs from
+Note that hunter's approach differs from a
```
Comment (ruslo) [inline docs/overview/automatic.rst]: `hunter` -> `Hunter`.

File: docs/overview/automatic.rst [hunk 5]
```diff
@@ -70,17 +64,23 @@ instead of:
...
 `requirements.txt-like approach <https://pip.readthedocs.org/en/1.1/requirements.html>`__
-(i.e. when external packages specified by separate file). This allow to avoid
-duplication of logic in many cases even if ``requirements.txt`` do downloads
-automatically too.
+(i.e. when external packages are specified in a separate file). This allows hunter to avoid
```
Comment (ruslo) [inline docs/overview/automatic.rst]: `hunter` -> `Hunter`

File: docs/overview/automatic.rst [hunk 6]
```diff
@@ -70,17 +64,23 @@ instead of:
...
+duplication of logic in many cases, even if the ``requirements.txt`` style approach also automatically downloads
+dependencies too.
 
-Imagine we have to specify dependencies in some kind of ``requirements.cmake``
-file and there is user's option:
+Imagine that we have to specify dependencies in some kind of ``requirements.cmake``
+file and there is a user option 'BUID_WITH_BAR':
```
Comment (ruslo) [inline docs/overview/automatic.rst]: Use
```
``BUILD_WITH_BAR``
```

File: docs/overview/automatic.rst (hunk 2 above)
Comment (ruslo) [reply docs/overview/automatic.rst]: I'm afraid no, at least that's what I remember.

Comment (kiates) [reply docs/overview/automatic.rst]: Yip, it was still grumpy.

File: docs/overview/automatic.rst (hunk 4 above)
Comment (ruslo) [reply docs/overview/automatic.rst]: ☝️

File: docs/overview/automatic.rst (hunk 5 above)
Comment (ruslo) [reply docs/overview/automatic.rst]: ☝️

**Pull request merged**